src/components/Card/index.js

```diff
@@ -18,7 +19,8 @@ class Card extends React.Component {
 
 Card.propTypes = {
   children: PropTypes.node,
-  className: PropTypes.string,
+  /** Padding `pl6 pr6` should be used for inner items that require space. */
```

Comment:
Pode acrescentar no inicio da descrição o proposito de ligar essa prop, algo como: "Turn this on to use the full width of the card. Respect the default padding (`pl6 pr6`) for elements that do not fill all full width." 

O que acha?

Reply:
Uma boa. Atualizado.